src/style.css

```
@@ -0,0 +1,64 @@
+a {
+  text-decoration: none;
+}
+
+ul {
```

```suggestion
.nav__list {
```
Try to avoid using tag-selector. Your list has class `nav__list`. Let's use it